src/pynwb/file.py

```diff
@@ -55,7 +56,8 @@ class Subject(NWBContainer):
             {'name': 'subject_id', 'type': str, 'doc': 'a unique identifier for the subject', 'default': None},
             {'name': 'weight', 'type': str, 'doc': 'the weight of the subject', 'default': None},
             {'name': 'date_of_birth', 'type': datetime, 'default': None,
-             'doc': 'datetime of date of birth. May be supplied instead of age.'})
+             'doc': 'datetime of date of birth. May be supplied instead of age.'},
+            {'name': 'strain', 'type': str, 'doc': 'the strain of the subject', 'default': None})
```

Comment:
Do we have a recommendation for how the strain should be specified. If so, it would be nice to add an example in the 'doc' .

Reply:
I updated this. Please review.